custom_components/tuya_local/devices/makegood_switch_dual_timer-energy.yaml

```diff
@@ -0,0 +1,87 @@
+name: Dual switch with timers and power monitoring
```

Comment:
I think name would be better shortened to just Dual switch.  The entity names will get added to the end of this, so it will become quite long and annoying to use in automations etc where the distinguishing part might scroll off the end of the field in the UI, or if you are manually editing be a lot to type.

Reply:
Sure, perhaps consider updating `simple_dual_switch_timer.yaml` and `simple_dual_switch_timer_v2.yaml`
```
name: Dual switch with timers
products:
...
```